Makefile

```diff
@@ -20,8 +20,8 @@ PACKAGE_NAME := universal-crossplane
 CROSSPLANE_REPO := https://github.com/upbound/crossplane.git
 # Tag corresponds to Docker image tag while commit is git-compatible signature
 # for pulling. They do not always match.
-CROSSPLANE_TAG := v1.5.0-rc.0.up.1
-CROSSPLANE_COMMIT := v1.5.0-rc.0-up.1
+CROSSPLANE_TAG := v1.6.0-rc.0.up.1.72.g4f292d31
```

Comment:
@muvaf do we need to use a non RC version? I believe our guarantees state that we will always use a release candidate on main (see 1 and 2) https://github.com/upbound/universal-crossplane/blob/main/VERSIONING.md

Reply:
Hmm you're right. Though I couldn't really figure out how I can get the CI action to run for this commit https://github.com/upbound/crossplane/releases/tag/v1.6.0-rc.0-up.1

It seems like `Makefile` didn't have the necessary fix back then https://github.com/upbound/crossplane/blob/6c780435a16545c9aa08af91de8852f1ad82461d/Makefile#L61

Reply:
Well, it actually publishes to `crossplane/crossplane` partially. @hasheddan do you know if there is any usage of `upboundci` user in `crossplane` org? It'd be great if we can take away all permissions of `upboundci` related to that org so that it cannot push there.

Reply:
@hasheddan I don't really have a great suggestion for this other than to `git checkout v1.6.0-rc.0-up.1`, fix Makefile to push to `upbound/crossplane` and run `make build.all`. How does it sound?

Reply:
@muvaf I think we can just tag `v1.6.0-rc.1-up.1` and use it (though it would be misaligned with upstream, not sure if our RC's need to stay in step)

Reply:
Hmm we can do that though it feels a bit misleading given the fact that you can't `git checkout v1.6.0-rc.1` in `crossplane/crossplane`

Reply:
@muvaf yeah I think you are right -- let's just stick with what you have here and update it to the new RC when we have an upstream one 👍🏻